modules/templates/helper.go

```diff
@@ -138,6 +138,20 @@ func NewFuncMap() template.FuncMap {
 			}
 			return user.Theme
 		},
+		"ThemeIcon": func(theme string) string {
+			if theme == "gitea-dark" {
+				return "octicon-moon"
+			} else if theme == "gitea-light" {
+				return "octicon-sun"
+			} else if theme == "gitea-auto" {
+				return "gitea-eclipse"
+			} else {
+				return "octicon-paintbrush"
+			}
```

Comment:
```suggestion
			switch theme {
	    case "gitea-dark":
		    return "octicon-moon"
	    case "gitea-light":
		    return "octicon-sun"
	    case "gitea-auto":
		    return "gitea-eclips"
	    default:
		    return "octicon-paintbrush"
	    }
```

Reply:
I like the switch way of writing, but gitea doesn't seem to have a normalized way of writing this, so it's the same with either one of them.

Reply:
I think I will go with the JSON config, making this switch unnecessary.

Reply:
->  Improve theme display #30671